packages/client/src/generation/TSClient/Model.ts

```diff
@@ -331,7 +331,7 @@ ${new PayloadType(this.outputType!, !this.dmmf.typeMap[model.name]).toTS()}
 ${new ModelDelegate(this.outputType!, this.dmmf, this.generator).toTS()}
 
 // Custom InputTypes
-${this.argsTypes.map(TS).join('\n')}
+${this.argsTypes.map((gen) => TS(gen)).join('\n')}
```

Comment:
I think you also need to change this in `Count.ts`, it has the same problem.

Reply:
see https://prisma-company.slack.com/archives/C032H34TACR/p1655733194404859?thread_ts=1655727931.637109&cid=C032H34TACR

Reply:
sure